arm-automation/2015-10-31/swagger/account.json

```diff
@@ -0,0 +1,578 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AutomationManagementClient",
```

Comment:
We don't put the word `Client` in the `title` anymore, since this describes the _service_ not the **client**